students/wachulec_maria/lesson_10_organizing_files/filling_in_the_gaps.py

```diff
@@ -0,0 +1,67 @@
+import os
+import re
+import shutil
+
+
+def search_prefix_in_files_names(prefix):
+    all_groups_list, list_of_numbers, names_list, result = [], [], [], None
```

Comment:
when we are creating multiple var in single line, they should be the same type. so here it would be more like:
```
all_groups_list, list_of_numbers, names_list = [], [], []
result = None
```